Badly packaged library - Wrapper for TIBCO Spotfire(R) - the Angular package

Hello,

I tried to follow instructions from https://www.npmjs.com/package/@tibco/spotfire-wrapper including creation of the new application from scratch - just making sure that Angular 9 packages are used instead of latest versions.

I installed the wrapper using npm install. I've also installed cdk, material and flex-layout.

Here are my dependencies:

```json
  "dependencies": {
    "@angular/animations": "~9.1.12",
    "@angular/cdk": "^9.2.4",
    "@angular/common": "~9.1.12",
    "@angular/compiler": "~9.1.12",
    "@angular/core": "~9.1.12",
    "@angular/flex-layout": "^9.0.0-beta.31",
    "@angular/forms": "~9.1.12",
    "@angular/material": "^9.2.4",
    "@angular/platform-browser": "~9.1.12",
    "@angular/platform-browser-dynamic": "~9.1.12",
    "@angular/router": "~9.1.12",
    "@tibco/spotfire-wrapper": "^0.10.0",
    "rxjs": "~6.6.2",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.901.12",
    "@angular/cli": "~9.1.12",
    "@angular/compiler-cli": "~9.1.12",
    "@angular/language-service": "~9.1.12",
    "@types/node": "^12.11.1",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.1.2",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.8.3"
  }
```

When I try to generate viewer component using `ng generate @tibco/spotfire-wrapper:dashboard --name MySpot` I get

> An unhandled exception occurred: Cannot find module 'C:\repos\spotfire-poc\node_modules\@tibco\spotfire-wrapper\schematics\ng-gen-dashboard\index'

When I try to include `import { SpotfireViewerModule } from '@tibco/spotfire-wrapper'` in my app.module.ts I get

> Cannot find module '@tibco/spotfire-wrapper' or its corresponding type declarations. ts(2307)

It looks like the published npm package is not packaged correctly.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,9 @@
+<a name="0.11.1"></a>
+## [0.11.1] (2020-08-10)
+
+### Features
+ * Restores the ability to install the NPM package. Issue #75.
+
 <a name="0.11.0"></a>
 ## [0.11.0] (2020-08-10)
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "spotfire-wrapper",
-  "version": "0.11.0",
+  "version": "0.11.1",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",

--- a/projects/spotfire-wrapper/CHANGELOG.md
+++ b/projects/spotfire-wrapper/CHANGELOG.md
@@ -1,3 +1,9 @@
+<a name="0.11.1"></a>
+## [0.11.1] (2020-08-10)
+
+### Features
+ * Restores the ability to install the NPM package. Issue #75.
+ 
 <a name="0.11.0"></a>
 ## [0.11.0] (2020-08-10)
 

--- a/projects/spotfire-wrapper/package.json
+++ b/projects/spotfire-wrapper/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@tibco/spotfire-wrapper",
-  "version": "0.11.0",
+  "version": "0.11.1",
   "private": false,
   "scripts": {
     "build": "../../node_modules/.bin/tsc -p tsconfig.schematics.json",
@@ -20,7 +20,10 @@
   },
   "schematics": "./schematics/collection.json",
   "author": "Nicolas Deroche <nderoche@tibco.com> (http://www.tibco.com)",
-  "keywords": ["TIBCO", "Spotfire"],
+  "keywords": [
+    "TIBCO",
+    "Spotfire"
+  ],
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",

--- a/projects/spotfire-wrapper/schematics/ng-add/index.ts
+++ b/projects/spotfire-wrapper/schematics/ng-add/index.ts
@@ -41,9 +41,9 @@ function addPackageToPackageJson(host: Tree, pkg: string, version: string, dev =
 export function ngAdd(_options: Schema): Rule {
     return (_host: Tree, context: SchematicContext) => {
         context.logger.log('info', `Installing @tibco/spotfire-wrapper`);
-        addPackageToPackageJson(_host, '@angular/material', '~8.0.2');
-        addPackageToPackageJson(_host, '@angular/cdk', '~8.0.2');
-        addPackageToPackageJson(_host, '@angular/flex-layout', '~8.0.0-beta');
+        addPackageToPackageJson(_host, '@angular/material', '^10.1.2');
+        addPackageToPackageJson(_host, '@angular/cdk', '~10.1.2');
+        addPackageToPackageJson(_host, '@angular/flex-layout', '~10.0.0-beta.32');
 
         const installTaskId = context.addTask(new NodePackageInstallTask());
 

